matplotlibrc.template

```diff
@@ -615,4 +615,5 @@
                                    ## On Windows use the full path since convert
                                    ## is also the name of a system tool.
 #animation.convert_args:           ## Additional arguments to pass to convert
-#animation.embed_limit : 20.0
+#animation.embed_limit : 20.0      ## Limit, in MB, of size of base64 encoded
```

Comment:
The limit is in MiB, not MB, and everything really should be fixed to say that.

Reply:
While technically, you are right, I tend not to change that. MB is still more well known and understandable to non computer science affine users than MiB.

Reply:
This is a tough one.  I have sympathy for the idea of helping maintain an international standard.  On the other hand, I had to look up if MiB was different than MB.